source/open-telemetry/tests/operation/atlas_search.yml

```
@@ -0,0 +1,139 @@
+description: operation atlas search
+schemaVersion: '1.26'
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeTracingMessages:
+        enableCommandPayload: true
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: operation-atlas-search
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: test
+
+runOnRequirements:
+  # Skip server versions without fix of SERVER-83107 to avoid error message "BSON field 'createSearchIndexes.indexes.type' is an unknown field."
+  # SERVER-83107 was not backported to 7.1.
+  - minServerVersion: "7.0.5"
+    maxServerVersion: "7.0.99"
+    topologies: [ replicaset, load-balanced, sharded ]
+    serverless: forbid
+  - minServerVersion: "7.2.0"
+    topologies: [ replicaset, load-balanced, sharded ]
+    serverless: forbid
+
+
+tests:
+  - description: atlas search indexes
+    operations:
+      - name: createSearchIndex
+        object: *collection0
+        arguments:
+          model: { definition: { mappings: { dynamic: true } } , type: 'search' }
+        expectError:
+          # This test always errors in a non-Atlas environment.  The test functions as a unit test  by asserting
```

```suggestion
          # This test always errors in a non-Atlas environment.  The test functions as a unit test by asserting
```

Consider a search/replace on 2+ spaces within comments to reduce unnecessary whitespace.